### PR TITLE
Add new 'workflows for Github actions'.

### LINKED ISSUE #4575: Modify runners version to 'ubuntu-latest'
## Describe the solution you'd like

Currently, the platform has several workflows, which perform various automatic actions, these workflows are launched through the`ubuntu-18.04` version of runners. 

According to several **warning** messages in different executions of the actions, we discovered that this version is being deprecated (see image). 

This information can also be found in the following link: https://github.com/actions/runner-images/issues/6002

It is necessary to update the version of the runners to the `ubuntu-latest` version. 

For more details on the available runner versions, please consult the following link: https://github.com/actions/runner-images#available-images

## Screenshots

image

### PR DESCRIPTION
**Pull request by matiasmoreno876**
# Description
Closes #4575

Hi team, in this PR I add new workflows for Github actions, these workflows were added in version [4.4 in this PR](https://github.com/wazuh/wazuh-kibana-app/pull/4344), the idea is to be able to work directly in the master branch with the workflows and update the version of the runners to 'ubuntu-latest'.

## Screenshots
image

### CODE REVIEW

**Review by Machi3mfl — APPROVED** [review]
LGTM

File: .github/workflows/wazuh-build-push-docker-action.yml [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  workflow_dispatch:
+    # Inputs the workflow accepts.
+    inputs:
+      build-manager-image:
+        type: boolean
+        description: 'Build manager image'
+        required: false
+      wazuh-manager-version: 
+        description: 'Wazuh manager version'
+        default: 'v4.3.4-rc1'
```
Comment (AlexRuiz7) [inline .github/workflows/wazuh-build-push-docker-action.yml:17]: Why this one uses an RC tag? This tag might no longer exist (in fact, it doesn't).

Also, I'd update the default version to the latest published one, which is 4.3.8, for every action.

**Review by AlexRuiz7 — CHANGES_REQUESTED** [review]
Some actions use RC tags to build the manager. These tags no longer exist.

File: .github/workflows/wazuh-build-push-docker-action.yml (hunk 1 above)
Comment (matiasmoreno876) [reply .github/workflows/wazuh-build-push-docker-action.yml]: Done

**Review by Mayons95 — APPROVED** [review]
LGTM

**Review by AlexRuiz7 — APPROVED** [review]
LGTM

**Pull request merged**